src/main/java/org/generationcp/middleware/domain/germplasm/AttributeDTO.java

```diff
@@ -0,0 +1,50 @@
+package org.generationcp.middleware.domain.germplasm;
+
+public class AttributeDTO {
```

Comment:
implement toString, hashCode and equals

Reply:
@vmaletta Please add @AutoProperty annotation when using pojomatics

Reply:
Ok